tools/cloud-build/daily-tests/tests/slurm-v5.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+deployment_name: "slurm-gcp-v5-{{ build }}"
```

Comment:
the reason I made it very short in my version is that we would not have clashes between running builds. Here, it seems that two builds running at the same time will clash (same cluster name).  Have you considered this risk?